packages/ui-components/src/components/Chart/PieChart.js

```diff
@@ -80,27 +87,51 @@ const makeCustomTooltip = ({ valueType, labelType }) => {
     }
 
     const data = payload[0].payload;
-    const { name, value, originalItem, fill } = data;
-    const metadata = originalItem[`${name}_metadata`];
+    const { name, fill } = data;
 
     return (
       <TooltipContainer>
         <Heading>{name}</Heading>
         <Item>
           <Box style={{ background: fill }} />
-          <Text>{formatDataValueByType({ value, metadata }, valueTypeForLabel)}</Text>
+          <Text>{getFormattedValue(viewContent, data)}</Text>
         </Item>
       </TooltipContainer>
     );
   };
 };
 
+const makeLabel = viewContent => {
+  return props => {
+    const { payload } = props;
+    return getFormattedValue(viewContent, payload.payload);
+  };
+};
+
 const chartColorAtIndex = (colorArray, index) => {
   return colorArray[index % colorArray.length];
 };
 
+const getHeight = (isExporting, isEnlarged) => {
+  if (isExporting) {
+    return 420;
+  }
+  return isEnlarged && isMobile() ? 320 : undefined;
+};
+
 export const PieChart = ({ viewContent, isExporting, isEnlarged, onItemClick, legendPosition }) => {
   const [activeIndex, setActiveIndex] = useState(-1);
+  const { presentationOptions, data } = viewContent;
+  // eslint-disable-next-line no-unused-vars
+  const [loaded, setLoaded] = useState(false);
+
+  // Delay rendering of the chart to fix an issue with the legend overlapping the chart.
+  // This is a work around for a recharts bug. @see https://github.com/recharts/recharts/issues/511
```

Comment:
But is it being used? I can't see `loaded` referenced anywhere (haven't checked out locally though)

Reply:
I've updated the comment to be a bit more accurate but all this does is triggers a re-render of the component and that does the trick. This change actually belongs to the WAI-2017 ticket but I've got it here and as I'm anticipating WAI-1027 getting into dev well before these changes.

Reply:
Aah right that makes sense!